lib/releases.nix

```diff
@@ -0,0 +1,22 @@
+# nixpkgs/nixos release information
+rec {
+
+  # formatting releases
+  asString = {
+    name
+  , releaseDate
+  }: "${name} (release date ${releaseDate})";
+
+  next = nixos-17-first;
```

Comment:
Shouldn't this be 17.09 (not 17.03)? If it's 17.03, there is no deprecation period for users of stable channels.